draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss

```diff
@@ -1,9 +1,8 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/shadow";
 @import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
-@import "~@kaizen/component-library/styles/border";
-@import "~@kaizen/component-library/styles/animation";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/animation";
 
 $heading-active-color: #f2f2f3;
```

Comment:
Don't know if we want to do this in this PR but wonder if we should get Jason to take a look to see if there's a comparable color-gray token?